.github/workflows/tests.yml

```
@@ -26,19 +26,18 @@ on:
 jobs:
   Tests:
     runs-on: ubuntu-20.04
-    timeout-minutes: 20
     strategy:
+      fail-fast: false
       matrix:
-          python-version: [3.7, 3.8, 3.9]
-          requirements-level: [pypi]
-          cache-service: [redis]
-          db-service: [postgresql10, postgresql13, mysql8]
+        python-version: ["3.9", "3.10", "3.11", "3.12"]
```

This was my mistake: I started adding all of them, but we agree that the matrix of tests is too large. We decided to keep 3.9 and 3.12, as we will upgrade InvenioRDM to 3.12, skipping the versions in between.
```suggestion
        python-version: [3.9, 3.12]
```